docs-ref-conceptual/azure-cli-vm-tutorial-4.md

```diff
@@ -12,7 +12,7 @@ ms.custom: devx-track-azurecli, seo-azure-cli
 keywords: azure cli show vm information, queries in azure cli
 ---
 
-# 4 - Get VM information with queries
```

Comment:
@daphnemamsft , even if Acrolinx doesn't like it, the `#4` shouldn't be replaced with `four`.  In this case, `4` is being used as a list number, not in a sentence.

Reply:
I have revised the number heading